components/emoji/emoji_list/emoji_list.tsx

```diff
@@ -151,18 +167,21 @@ export default class EmojiList extends React.PureComponent {
                     key='loading'
                     className='backstage-list__item backstage-list__empty'
                 >
-                    <td colSpan='4'>
+                    <td colSpan={4}>
                         <LoadingScreen key='loading'/>
                     </td>
                 </tr>,
             );
-        } else if (this.props.emojiIds.length === 0 || (searchEmojis && searchEmojis.length === 0)) {
+        } else if (
+            this.props.emojiIds.length === 0 ||
+            (searchEmojis && searchEmojis.length === 0)
```

Comment:
The new `searchEmojis = []` modifies the behavior here, `null`is false but `[]` is true, so this is not exactly the same behavior.

Reply:
After reviewing a bit more the code, looks like it should have the same behavior. I'm not 100% sure, but looks like, if you change the search, and end up failing the search or returning no data, you end up in the same state, so... I guess we going to achieve the correct behavior or expose another corner case.

Reply:
Hello @jespino Thanks for pointing it out :smile: . I tried to think about it and you're right this does change the behaviour, as `searchEmojis` will now be true. But I'm guessing earlier it needs to check that because it was null and pushing to null will result in an error. But if `searchEmojis` is an `[]` then it should have no problem in pushing to an empty array. Infact we don't need the first condition at all. However, just like you I'm also 100% sure about this too :stuck_out_tongue: . Do you think we should get a third opinion? Let me know